core/src/main/scala/com/nvidia/spark/rapids/tool/profiling/ProfileClassWarehouse.scala

```diff
@@ -202,38 +204,106 @@ class SQLExecutionInfoClass(
   }
 }
 
-case class SQLAccumProfileResults(appIndex: Int, sqlID: Long, nodeID: Long,
-    nodeName: String, accumulatorId: Long, name: String, min: Long, median:Long,
-    max: Long, total: Long, metricType: String, stageIds: String) extends ProfileResult {
-  override val outputHeaders = Seq("appIndex", "sqlID", "nodeID", "nodeName", "accumulatorId",
-    "name", "min", "median", "max", "total", "metricType", "stageIds")
+case class SQLAccumProfileResults(
```

Comment:
It is hard to tell if there are changes in that class. Is this only for sake of code formatting?
If so, it will be nice to put a PR comment on the code change to make it easy to distinguish between format changes Vs. argument changes.
That's probably something we need to address in our PR improvement process.

Reply:
Yes, I changed this for better readability.
I put a comment in PR description about this, but yes, I should put a comment in the code changes too.
Thanks!